### DISCUSSION TITLE
Will it be possible to support multiple extensions of the same type?

### DISCUSSION THREAD

**Discussion by MikhailGordienk** [eclipse-edc/Connector#743] [Q&A] [answered]
Will it be possible to support multiple extensions of the same type (ex. multiple Vault types)?

**Comment by jimmarino** [comment, marked as answer]
This is currently supported in two ways:

1. One Vault implementation can be substituted for another by configuring the appropriate extensions (e.g. include the Vault module you want in your build)

2. In case you want to store secrets in different places, implement a multiplexing vault extension that delegates to different Vault backends
